lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -291,5 +296,24 @@ export type PerformanceEvent = {
      *
      * @type {?boolean}
      */
-    isNativeBroker?: boolean
+    isNativeBroker?: boolean,
+
+    /**
+     * The following are fields that may be emitted in native broker scenarios
+     */
+    extensionId?: string,
+    extensionVersion?: string
+    mats_broker_version?: string;
+    mats_account_join_on_start?: string;
+    mats_account_join_on_end?: string;
+    mats_device_join?: string;
+    mats_prompt_behavior?: string;
+    mats_pi_error_code?: number;
+    mats_ui_visible?: boolean;
+    mats_silent_code?: number;
+    mats_silent_bi_sub_code?: number;
+    mats_silent_message?: string;
+    mats_silent_status?: number;
+    mats_http_status?: number
+    mats_http_event_count?: number;
```

Comment:
Do these need to be snake_cased? Or can they match the casing of the other properties?

Reply:
Nope - was originally trying to match the keys coming through from MATS to make implementation cleaner but that's no longer the case so I'll change this